Frontend/src/index.css

```diff
@@ -1,4 +1,4 @@
 body {
-  margin: 0px;
+  margin: -10px;
```

Comment:
margin: -10px feels a little odd. Is margin:0 with padding: 10px works for your styling?

Reply:
I used this styling in order to fix the extra margin which caused a border around all my elements for some reason. When margin is at 0px, there is still a border.